lib/jekyll/frontmatter_defaults.rb

```diff
@@ -131,18 +136,18 @@ def valid?(set)
     # new_scope - the new scope hash
     #
     # Returns true if the new scope has precedence over the older
-    def has_precedence?(old_scope, new_scope)
+    def precedence?(old_scope, new_scope)
```

Comment:
@ayastreb this will have to be reverted back to it's original name, it's an API change that we cannot accept in a minor version.  Feel free to "XXX" it for change in 4.0 but we might have people who somehow rely on it in some way shape or form, so we should keep it as is and tell Rubocop to silence it for now.